src/gesture.js

```diff
@@ -94,8 +97,11 @@ export class Gestures {
     /** @private {?GestureRecognizer} */
     this.eventing_ = null;
 
+    const win = element.ownerDocument.defaultView;
+    const passiveSupported = supportsPassiveEventListener(toWin(win));
+
     /** @private {boolean} */
-    this.shouldNotPreventDefault_ = shouldNotPreventDefault;
+    this.shouldNotPreventDefault_ = shouldNotPreventDefault || passiveSupported;
```

Comment:
This was needed because in passive mode, calling `preventDefault` is invalid, and logs a warning to the console.